src/main/java/com/conveyal/datatools/editor/controllers/api/AgencyController.java

```diff
@@ -73,6 +76,8 @@ public static Object createAgency(Request req, Response res) {
         } catch (Exception e) {
             e.printStackTrace();
             halt(400);
+        } finally {
+            if (tx != null) tx.rollbackIfOpen();
```

Comment:
I was using `rollback` in functions handling `get` requests, but maybe `rollbackIfOpen` is always the way to go.